app/jenkins-for-jira-ui/src/App.tsx

```diff
@@ -94,7 +98,7 @@ const App: React.FC = () => {
 		};
 	}, []);
 
-	if (!history) {
+	if (!history || isFetchingFlag) {
```

Comment:
When testing locally last night I noticed that sometimes the old UI would render briefly sometimes. Added this additional check to stop that... and don't worry, I tested that both configured and not configured states still work 🙃